tests/conftest.py

```diff
@@ -21,6 +21,8 @@
 ACCEPTABLE_FAILURE_RATE = 0
 
 core.settings.disable_off_screen_rendering()
```

Comment:
@PProfizi why do you need the environment varibales? core.settings.disable_off_screen_rendering() serves the same purpose

Reply:
@cbellot000 I know, but for Linux it seems it did not work, it kept freezing at the doctest for plotter.show_figure